src/app-layout/__integ__/app-layout-split-panel.test.ts

```diff
@@ -313,6 +317,21 @@ describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)(
     })
   );
 
+  test(
+    'side panel stays in the viewport when scrolling down',
+    setupTest(async page => {
+      await page.openPanel();
+      await page.switchPosition('side');
+      await expect(
+        page.isDisplayedInViewport(wrapper.findSplitPanel().findOpenPanelSide().toSelector())
+      ).resolves.toEqual(true);
+      await page.windowScrollTo({ top: 500 });
+      await expect(
```

Comment:
Does this assertion fail without the fix? `findOpenPanelSide` returns the entire split panel, and `isDisplayedInViewport` will return `true` even if the element is only partially visible: https://webdriver.io/docs/api/element/isDisplayedInViewport/

Reply:
Passes, so I would not call this a real regression test. Do you have any suggestions for a better test?

Reply:
You could take the vertical position of the panel before and after scrolling and assert that it is the same.

Alternatively, you could verify that some internal element at the top (such as the header) is visible instead of the entire panel. But this approach is more dependent on element sizes so it feels a bit more fragile.

Reply:
Thanks for the suggestion. Now I got the test failing before and working after